### PR TITLE
Adding ingots

### PR DESCRIPTION
**Pull request by siaynoq**
Alloys will be next after this PR

### CODE REVIEW

**Comment by siaynoq** [comment]
Sure thing, it really does look/sound better like that

File: pages/Tin-Ingot.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+## Obtaining
+Smelting one [Tin Dust](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Tin-Dust) in a [Smeltery](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Smeltery) or an [Electric Ingot Factory](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Electric-Ingot-Factory) yields one Tin Ingot.<br>
+
+Tin Ingots can be turned back to [Tin Dust](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Tin-Dust) by an [Electric Ingot Pulverizer](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Electric-Ingot-Pulverizer)
+
+## Usage
+Tin Ingots are used in the following recipe:
+* [Tin Can](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Tin-Can)
+
```
Comment (TheBusyBiscuit) [inline pages/Tin-Ingot.md:10]: Should probably also mention that it is very important for alloys such as Bronze.

File: pages/Zinc-Ingot.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+## Obtaining
+Smelting one [Zinc Dust](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Zinc-Dust) in a [Smeltery](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Smeltery) or an [Electric Ingot Factory](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Electric-Ingot-Factory) yields one Zinc Ingot.<br>
+
+Zinc Ingots can be turned back to [Zinc Dust](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Zinc-Dust) by an [Electric Ingot Pulverizer](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Electric-Ingot-Pulverizer)
+
+## Usage
+
+Zinc Ingots are used in the following recipes:
+* [Battery](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Battery)
+* [GPS Teleporter Pylon](https://github.com/TheBusyBiscuit/Slimefun4/wiki/GPS-Teleporter-Pylon)
```
Comment (TheBusyBiscuit) [inline pages/Zinc-Ingot.md:11]: Also here: Alloys like Brass

File: pages/Tin-Ingot.md (hunk 1 above)
Comment (siaynoq) [reply pages/Tin-Ingot.md:10]: For Bronze, Tin Dust is used, not the Ingot version

File: pages/Zinc-Ingot.md (hunk 2 above)
Comment (siaynoq) [reply pages/Zinc-Ingot.md:11]: Likewise, Brass needs Zinc Dust, not Ingot.
I was thinking about some sort of "overall Dust-Ingot-Alloy diagram" within an article, which show all the relations between them - maybe we could put some "related" links here to help people find what they want?

**Pull request merged**